<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BufferWidget</class>
 <widget class="QWidget" name="BufferWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QScrollBar" name="scrollBarVertical">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar:vertical
{
    background-color: transparent;
    width: 16px;
}

QScrollBar::handle:vertical
{
    background-color: rgb(46, 58, 62);
    min-height: 20px;
    border-radius: 3px;
	margin: 2px;
}

QScrollBar::handle:vertical:pressed
{
    background-color: rgb(58, 70, 74);
    min-height: 20px;
    border-radius: 3px;
	margin: 2px;
}

QScrollBar::handle:vertical:hover:!pressed
{
    background-color: rgb(50, 62, 66);
    min-height: 20px;
    border-radius: 3px;
	margin: 2px;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
	background-color: transparent;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical
{
	background-color: transparent;
	width: 0px;
	height: 0px;
}

QScrollBar:horizontal
{
    background-color: transparent;
    height: 16px;
}

QScrollBar::handle:horizontal
{
    background-color: rgb(46, 58, 62);
    min-width: 20px;
    border-radius: 3px;
	margin: 2px;
}

QScrollBar::handle:horizontal:pressed
{
    background-color: rgb(58, 70, 74);
    min-width: 20px;
    border-radius: 3px;
	margin: 2px;
}

QScrollBar::handle:horizontal:hover:!pressed
{
    background-color: rgb(50, 62, 66);
    min-width: 20px;
    border-radius: 3px;
	margin: 2px;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
	background-color: transparent;
}

QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal
{
	background-color: transparent;
	width: 0px;
	height: 0px;
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QScrollBar" name="scrollBarHorizontal">
     <property name="styleSheet">
      <string notr="true">QScrollBar:vertical
{
    background-color: transparent;
    width: 16px;
}

QScrollBar::handle:vertical
{
    background-color: rgb(46, 58, 62);
    min-height: 20px;
    border-radius: 3px;
	margin: 2px;
}

QScrollBar::handle:vertical:pressed
{
    background-color: rgb(58, 70, 74);
    min-height: 20px;
    border-radius: 3px;
	margin: 2px;
}

QScrollBar::handle:vertical:hover:!pressed
{
    background-color: rgb(50, 62, 66);
    min-height: 20px;
    border-radius: 3px;
	margin: 2px;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
	background-color: transparent;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical
{
	background-color: transparent;
	width: 0px;
	height: 0px;
}

QScrollBar:horizontal
{
    background-color: transparent;
    height: 16px;
}

QScrollBar::handle:horizontal
{
    background-color: rgb(46, 58, 62);
    min-width: 20px;
    border-radius: 3px;
	margin: 2px;
}

QScrollBar::handle:horizontal:pressed
{
    background-color: rgb(58, 70, 74);
    min-width: 20px;
    border-radius: 3px;
	margin: 2px;
}

QScrollBar::handle:horizontal:hover:!pressed
{
    background-color: rgb(50, 62, 66);
    min-width: 20px;
    border-radius: 3px;
	margin: 2px;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
	background-color: transparent;
}

QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal
{
	background-color: transparent;
	width: 0px;
	height: 0px;
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="dummyWidget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
